packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -33,7 +33,7 @@ import 'framework.dart';
 /// platform view iff it won the arena. Specific gestures that should be dispatched to the platform
 /// view can be specified in the `gestureRecognizers` constructor parameter. If
 /// the set of gesture recognizers is empty, a gesture will be dispatched to the platform
-/// view iff it was not claimed by any other gesture recognizer.
```

Comment:
This is a short-hand for "if and only if" as used in math: https://en.wikipedia.org/wiki/If_and_only_if. It's not a typo :)